src/app/services/congregation/congregation.service.ts

```diff
@@ -5,13 +5,17 @@ import { HttpService } from '../http/http.service';
 
 @Injectable()
 export class CongregationService {
-  constructor(private httpService: HttpService) { }
+  constructor(private httpService: HttpService) {}
 
   public getCongregation(): Observable<Congregation[]> {
-    return this.httpService.get(`/v1.0.0/congregation`).map(congregationResponse => {
-      return congregationResponse.map(c => {
-        return new Congregation(c.name, c.id)
-      })
-    })
+    return this.httpService
+      .get(`/v1.0.0/congregation`)
+      .map(congregationResponse => {
+        return congregationResponse
+          .filter(c => c.id !== 2)
```

Comment:
@janssejl I gotta be a stickler here. that `!==2` is kind of buried. It's also not super obvious WHY that !== 2 is there. I'd like to put either a comment in there explaining it or an enum (comment is probably better?)
```suggestion
          .filter(c => c.id !== 2) // filters out 'I Don't Attend' as an option
``` 
If we get any more requests to filter this list out we should probably put that business logic on the back end and not the front end.

Reply:
Done.